src/linker/Linker/Driver.cs

```diff
@@ -864,12 +866,12 @@ protected bool AddMarkHandler (Pipeline pipeline, string arg)
 			return true;
 		}
 
-		bool TryGetCustomAssembly (ref string arg, out Assembly assembly)
+		bool TryGetCustomAssembly (ref string arg, [NotNullWhen (true)] out Assembly? assembly)
 		{
 			assembly = null;
 			int pos = arg.IndexOf (",");
 			if (pos == -1)
-				return true;
```

Comment:
I guess I just want to understand this one better, what is a custom assembly for driver purposes and what is the format that the coma seems to be important